src/common/helpers.js

```diff
@@ -359,3 +359,23 @@ export function numberFormatter(num, digits = 0) {
   }
   return (num / si[i].value).toFixed(digits).replace(rx, '$1') + si[i].symbol;
 }
+
+/**
+ * Subscribe an email to CCDL Emails
+ *
+ * @param {*} EmailAddress
+ */
+
+export const subscribeToMailingList = async email => {
+  const portalId = '5187852';
+  const formGuid = 'ea5b970d-c22f-4b42-b569-26f84b1cea2f';
+  const formUrl = `https://api.hsforms.com/submissions/v3/integration/submit/${portalId}/${formGuid}`;
+  try {
+    await Ajax.post(formUrl, {
+      fields: [{ name: 'email', value: email }],
+    });
+  } catch (err) {
+    // Show error message?"
```

Comment:
We should log the failure to somewhere, maybe sentry?

Reply:
we talked about this before... this is how it is currently implemented but im open to telling sentry.. is there any problem with passing the email or we could pass the dataset and then manually subscribe them to the mailing list if it fails, but I think it's very unlikely to fail unless maybe they have a weird internet connection drop

Reply:
yes, sending the dataset id should be enough. Hopefully, this never fails

Reply:
I'm not really sure how we would escalate exceptions to Sentry without triggering an error boundary capture, do you have any direction for this? Also let's make it a new ticket because this isn't really a change specific to this PR it's just moving the same logic we had so it can be shared.